docs-source/content/samples/simple/domains/soa-domain/_index.md

```diff
@@ -111,11 +113,11 @@ into the target directory, maintaining the original directory hierarchy.
 
 The default domain created by the script has the following characteristics:
 
-* An Administration Server named `admin-server` listening on port `7001`.
+* An Administration Server named `AdminServer` listening on port `7001`.
 * A configured cluster named `soa_cluster-1` of size 5.
-* Two Managed Servers, named `soa-server1` and `soa-server2`, listening on port `8001`.
+* Two Managed Servers, named `soa_server1` and `soa_server2`, listening on port `8001`.
```

Comment:
It may be worth noting that server names such as "soa_server1" will become hostnames & listen addresses using "soa-server1" because we have to normalize to the k8s rules.

Reply:
Ok. Please let me know if you want me to revert the server names to have - instead of _
In that case I need to update the create-domain-inputs.yaml as well. Please confirm.

Reply:
You don't need to make a change.  You may consider putting a note about the disconnect between the server name and the host name.